File: Laboratorio03/cpp/037arcCrearArchivo.cpp
#include <iostream>
#include <list>
#include <cstdlib>
#include <algorithm>
#include <thread>
#include <chrono>
#include <map>
#include <fstream>
using namespace std;
int main() {
    string nombreArchivo = "archivo.txt";
    try {
        ofstream archivo(nombreArchivo);
        archivo.close();
    } catch (const exception& exception) {
        cerr << "Ocurrio un error al crear el archivo " << nombreArchivo << endl;
    }
    return 0;
}

File: Laboratorio03/python/037arcCrearArchivo.py
import os

nombreArchivo = 'miArchivo.txt'

os.system("cls")

try:
    archivo = open(nombreArchivo, 'w')
    archivo.close()
    print('Se ha creado el archivo:', nombreArchivo, '\n')
except IOError:
    print('Ocurrió un error al crear el archivo:', nombreArchivo)
